sway-core/src/parse_tree/declaration/type_parameter.rs

```diff
@@ -59,5 +59,5 @@ impl TypeParameter {
 
 #[derive(Debug, Clone, Eq, PartialEq, Hash)]
 pub(crate) struct TraitConstraint {
-    pub(crate) name: Ident,
+    pub(crate) name: CallPath,
```

Comment:
Would you be able to change this from `name` to `call_path`?

Reply:
Done!